CHANGELOG.md

```diff
@@ -26,8 +26,8 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
   image sizes on mobile carousel.
 - Added `u-link__disabled` utility class for styling disabled links.
 - Added `/careers/working-at-cfpb/` page.
-- Added block templates for current openings, job app process,
-  linkedin info, and provide feedback link.
+- Added block templates for linkedin info, provide feedback link,
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/b713d41a385b210f5bd2fd3a0ca481ab8d0a405e

> Should it be the same for the template name?

No, this should be okay. Best to keep all templates filenames lowercase.